docs/user-guide/kubernetes.md

```diff
@@ -67,6 +67,64 @@ To deploy Træfɪk to your cluster start by submitting the deployment to the clu
 ```sh
 kubectl apply -f examples/k8s/traefik.yaml
 ```
+### Role Based Access Control configuration (optional)
+
+Kubernetes introduces [Role Based Access Control (RBAC)](https://kubernetes.io/docs/admin/authorization/) in 1.6+ to allow fine-grained control
+of Kubernetes resources and api.
+
+If your cluster is configured with RBAC, you need to authorize Træfɪk to use
+kubernetes API using ClusterRole, ServiceAccount and ClusterRoleBinding resources:
+
+```yaml
```

Comment:
Unfortunately, it [does not seem to be supported](https://github.com/github/markup/issues/172#issuecomment-33241601).

Personally, I don't mind the flow break. After all, that's what hyperlinking is all about. :-)

That said, I'm not overly enthusiastic on this point. A bit of duplication is acceptable for me here.